### DISCUSSION TITLE
Smart after core update_ Scripts error, how to fix it

### DISCUSSION THREAD

**Discussion by zjgzh001** [azerothcore/azerothcore-wotlk#16351] [General]
Smart after core update_ Scripts error, how to fix it
Call action_ Error in type 85
SmartAIMgr: Not handled action_type(85), event_type(62), Entry 90007 SourceType 0 Event 396, skipped.
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/133203282/2438347f-fda3-47ab-8a12-4d2ce6c52fa1)

**Comment by Nyeriah** [comment]
Clean your database or revert your custom changes, or fix your smartscripts, we don't have such errors on the last revision.

**Reply by zjgzh001** [reply]
Hello, there was no such error in the previous version, but now it appears a bit large. Can you please advise me on how to fix the smart script? I guess it's because the calling code for 85 does not exist that this error occurs
